jenkins-scripts/lib/_homebrew_cleanup.bash

```diff
@@ -15,6 +15,7 @@ export HOMEBREW_UPDATE_TO_TAG=1
 ${BREW_BINARY} up || { restore_brew && ${BREW_BINARY} up ; }
 
 # Clear all installed homebrew packages, links, taps, and kegs
+${BREW_BINARY} list --formula | wc || { restore_brew && ${BREW_BINARY} list --formula | wc; }
```

Comment:
What is the goal of using the pipe with `wc`?

Reply:
it's just trying to reduce the console spam, since `brew list` can generate dozens of lines, and we are potentially running it twice. `wc` would summarize the size of the output. Should I add a comment to that effect to explain its use?

Reply:
I see. I assume that you are somehow interested in the summary output or could we just use >/dev/null as usual? I'm worried about using the pipe could make the `||` not to work as expected:
```bash
❯ false | wc || { echo foo; }
      0       0       0
❯ true | wc || { echo foo; }
      0       0       0
```

Reply:
ok, I'll pipe to /dev/null instead

Reply:
https://github.com/ignition-tooling/release-tools/pull/679/commits/b6489c0e6a1557e61ec944942563b490086efd19